cli/cmd/root.go

```diff
@@ -108,6 +108,20 @@ var rootCmd = &cobra.Command{
 				return err
 			}
 			setMountPointsForFamiliesInput(mountPoints, config)
+		} else {
+			// Check if volume already mounted
+			scanPath := "/mnt/snapshot"
```

Comment:
It would be good to define this as a constant somewhere

Reply:
I can add this as a constant for the cli and for the provider :)

Reply:
updated with https://github.com/openclarity/vmclarity/pull/462/commits/c99f61c9a6bf68abaf22daa3625e6e2c20fc23e2